cpp/arcticdb/version/version_map.hpp

```diff
@@ -107,7 +107,7 @@ class VersionMapImpl {
      */
     using MapType =  std::map<StreamId, std::shared_ptr<VersionMapEntry>>;
 
-    static constexpr uint64_t DEFAULT_CLOCK_UNSYNC_TOLERANCE = ONE_SECOND * 2;
+    static constexpr uint64_t DEFAULT_CLOCK_UNSYNC_TOLERANCE = ONE_MILLISECOND * 200;
```

Comment:
This is from the last commit. Pasting description below:
```
Before this change version_map's reload_interval was equal to the
unsync_tolerance. This means that we would ~always miss the cache.

The check for outdated cache is as follows:
- When reading into cache set `last_reload_time = now -
  unsync_tolerance`
- When checking if cache is outdated we check whether `now -
  last_reload_time > reload_interval

This change updates the unsync tolerance to 200ms. This will mean the
cache is valid for 1.8s

Tests uncovered that we were not flushing the version_map when clearing
the storage. This change fixes this as well.
```

Reply:
I don't understand the motivation for the "clock unsync tolerance" - it's being applied to times from this process' clock and only being held in memory. How can we have clock skew within a single process? Any memories @willdealtry?

Reply:
AFAICT this unsync tolerance is somehow a tolerance WRT this process' own clock, which theoretically can jump around a little bit, but I don't think reducing the tolerance to 200ms is very worrying.

Reply:
Agreed